server/routers/jobs_router.py

```
@@ -270,249 +433,155 @@ def load_fiche_metier():
+                    emploiReglemente = fiche.get("emploiReglemente"),
+                    nb_offre = fiche.get("nb_offre"),
+                    liste_salaire_offre = fiche.get("liste_salaire_offre")
+                                )
                 try:
                     session.add(fiche_insert)
-                except Exception as e:
-                    # On loggue l'erreur avec la stack trace complète et l'ID de la fiche
-                    logger.exception(f"Erreur lors de l'ajout de la fiche {fiche.get('code')}: {e}")
-                    # On continue la boucle pour traiter les fiches suivantes
-                    continue
+                except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)